rastervision/backend/torch_utils/chip_classification/data.py

```diff
@@ -43,7 +44,11 @@ def __len__(self):
 
 
 def build_databunch(data_dir, img_sz, batch_sz, class_names, augmentors):
-    num_workers = 4
+    # To avoid memory errors
```

Comment:
What happened without this check? I can see that using multiple GPUs on one instance means that there could be fewer CPUs available per GPU and so `num_workers` might need to be lower, but I don't think we want to set this 0 unless we have to. I've had situations where things froze when using `num_workers > 0`, but it's best to avoid setting it to zero.

Reply:
Without this check I got memory errors. Even for super small batch sizes (`batch_size=16`, Nvidia P40 with 22GB of memory, using resnet50). My guess is that the multiple processes try to allocate the same bit of memory on the GPU. On the documentation page of [torch.utils.data](https://pytorch.org/docs/stable/data.html) I came across the red box (1/3rd down the page, same as in the below comment), so I tried with `num_workers=0`, and that seemed to remove the memory error.